README.md

```diff
@@ -1,53 +1,51 @@
-## Project name
+## gcloud-golang-todos
 
-A description of what this project does and who it serves.
+> [TodoMVC](http://todomvc.com) backend using [gcloud-golang](//github.com/GoogleCloudPlatform/gcloud-golang).
 
-Include authorship, support contact and release information.
 
+### Prerequisites
 
-## Project setup, installation, and configuration
+1. Create a new cloud project on [console.developers.google.com](https://console.developers.google.com)
```

Comment:
Step 0: set up a [Go installation](https://golang.org/doc/install) and [workspace](https://golang.org/doc/code.html).